backend/src/main/java/NumericalRangeQuestion.java

```diff
@@ -0,0 +1,79 @@
+import javax.persistence.Entity;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Numerical type question
+ */
+
+@Entity
+public class NumericalRangeQuestion extends Question {
+
+    private List<Float> answers = new ArrayList<>();
+    protected int lowerBound;
+    protected int upperBound;
+
+    /**
+     * Default constructor
+     */
+    public NumericalRangeQuestion() {
+        super();
+    }
+
+    /**
+     * Constructor with specified question
+     * @param question the question text
+     */
+    public NumericalRangeQuestion(String question, int min, int max) {
```

Comment:
floats for upperBound and lowerBound, change parameter names to also be upperBound and lowerBound